packages/tuono/src/build/config/normalize-config.ts

```diff
@@ -71,6 +71,7 @@ export const normalizeConfig = (config: TuonoConfig): InternalTuonoConfig => {
   return {
     server: {
       host: config.server?.host ?? 'localhost',
+      origin: config.server?.origin ?? 'http://localhost:3000',
       port: config.server?.port ?? 3000,
     },
```

Comment:
Isn't this field ok to be `undefined`?
```suggestion
    server: {
      host: config.server?.host ?? 'localhost',
      origin: config.server?.origin,
      port: config.server?.port ?? 3000,
    },
```

Reply:
It was not due to the `Required` and `NonNullable` in `Required<NonNullable<TuonoConfig['server']>>` in `packages/tuono/src/build/types.ts`, however both `Required` and `NonNullable` have now been removed to allow optional values in the internal config.

I have removed the default value for origin here now.